lib/graphql/schema/member/has_arguments.rb

```diff
@@ -38,39 +38,6 @@ def argument(*args, **kwargs, &block)
           end
           arg_defn = self.argument_class.new(*args, **kwargs, &block)
           add_argument(arg_defn)
-
-          if self.is_a?(Class) && !method_defined?(:"load_#{arg_defn.keyword}")
-            method_owner = if self < GraphQL::Schema::InputObject || self < GraphQL::Schema::Directive
-              "self."
-            elsif self < GraphQL::Schema::Resolver
-              ""
-            else
-              raise "Unexpected argument owner: #{self}"
-            end
-            if loads && arg_defn.type.list?
-              class_eval <<-RUBY, __FILE__, __LINE__ + 1
-              def #{method_owner}load_#{arg_defn.keyword}(values, context = nil)
-                argument = get_argument("#{arg_defn.graphql_name}", context || self.context)
-                (context || self.context).query.after_lazy(values) do |values2|
-                  GraphQL::Execution::Lazy.all(values2.map { |value| load_application_object(argument, value, context || self.context) })
-                end
-              end
-              RUBY
-            elsif loads
-              class_eval <<-RUBY, __FILE__, __LINE__ + 1
-              def #{method_owner}load_#{arg_defn.keyword}(value, context = nil)
-                argument = get_argument("#{arg_defn.graphql_name}", context || self.context)
-                load_application_object(argument, value, context || self.context)
-              end
-              RUBY
-            else
-              class_eval <<-RUBY, __FILE__, __LINE__ + 1
-              def #{method_owner}load_#{arg_defn.keyword}(value, _context = nil)
-                value
-              end
-              RUBY
-            end
-          end
```

Comment:
I thought, "maybe I could remove these methods and put a default implementation in the code instead."  I tracked down where these are called and found that there's _already_ a default implementation there: 

https://github.com/rmosolgo/graphql-ruby/blob/91f33818329b4503fa0959b0d2939720cff46639/lib/graphql/schema/argument.rb#L327-L336

There's some chance this will break calls to `super`. In that case I think the thing to do is manually call this code there (`load_application_object(...)`). And I guess it hasn't come up yet -- there's no test for calling `super`.